docs/fides/docs/language/resources/dataset.md

```diff
@@ -109,6 +113,19 @@ dataset:
             description: User's Email
             data_categories:
               - user.provided.identifiable.contact.email
+          - name: phone
+            description: User's phone numbers
+            data_categories:
+              - user.provided.identifiable.contact.phone_number
+            fields:
+              - name: mobile
+                description: User's mobile phone number
+                data_categories:
+                  - user.provided.identifiable.contact.phone_number
+              - name: home
+                description: User's home phone number
+                data_categories:
+                  - user.provided.identifiable.contact.phone_number
```

Comment:
Just so I understand, this model we are creating doesn't preserve what the original structure of the data was right? 

For instance it seems like this model could correspond to two different inputs:
```
{
  "person": {
    "phone": {
      "home": {
        "name": "home",
        "number": "123456"
      },
      "mobile": {
        "name": "mobile",
        "number": "98765"
      }
    }
  }
}
```

```
{
  "person": {
    "phone": [
      {
        "name": "home",
        "number": "123456"
      },
      {
        "name": "mobile",
        "number": "98765"
      }
    ]
  }
}
```

So a nested field could correspond to a list or object child?

Reply:
I think that is where I am at so far, thank you for clarifying it so well above - one of the things that I was struggling with was confidence in being able to see all of the patterns that may apply and allow for it until we can test it more in depth. Where I also got a little twisted was if/when a field is used as a dumping ground for many different types of data and can contain a host of different levels or attributes of varying bits of information